arches_lingo/src/arches_lingo/components/concept/ConceptHeader.vue

```diff
@@ -0,0 +1,179 @@
+<script setup lang="ts">
+import { inject, onMounted, ref, watch } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import { useToast } from "primevue/usetoast";
+import Button from "primevue/button";
+import Divider from "primevue/divider";
+import {
+    DEFAULT_ERROR_TOAST_LIFE,
+    ERROR,
+    systemLanguageKey,
+    VIEW,
+} from "@/arches_lingo/constants.ts";
+
+import { fetchLingoResource } from "@/arches_lingo/api.ts";
+import { extractDescriptors } from "@/arches_lingo/utils.ts";
+
+import ResourceInstanceMultiSelectWidget from "@/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/ResourceInstanceMultiSelectWidget.vue";
+
+import type {
+    ConceptHeader,
+    ConceptClassificationStatusAliases,
+    ResourceInstanceResult,
+    DataComponentMode,
+} from "@/arches_lingo/types";
+import type { Language } from "@/arches_vue_utils/types";
+
+const toast = useToast();
+const { $gettext } = useGettext();
+const systemLanguage = inject(systemLanguageKey) as Language;
+
+const concept = ref<ResourceInstanceResult>();
+
+const props = defineProps<{
+    mode: DataComponentMode;
+    sectionTitle: string;
+    componentName: string;
+    graphSlug: string;
+    resourceInstanceId: string;
+    nodegroupAlias: string;
+}>();
+
+const data = ref<ConceptHeader>();
+
+watch(
```

Comment:
you shouldn't need a watch here, instead this could be a function that's run after the `await` in the onMounted script